packages/cli/src/project/handlebars.ts

```diff
@@ -0,0 +1,33 @@
+import * as changeCase from 'change-case';
+import Handlebars from 'handlebars';
+
+import { IProject } from './project';
+
+
+export class HandlebarsTemplate {
+    static _runtimeOptions: Handlebars.RuntimeOptions = {
+        helpers: {
+            capitalize: (text: string) => {
+            if (!text) return '';
+            return changeCase.capitalCase(text);
```

Comment:
can't tell if the formatting is weird here or if its just github showing it that way

Reply:
oh, i thought the linter would've fixed it. Pushed a fix